pkg/cmd/fetch/fetch.go

```diff
@@ -594,6 +598,107 @@ func newCmdChainguardOSV() *cobra.Command {
 	return cmd
 }
 
+func newCmdCiscoJSON() *cobra.Command {
+	options := &base{
+		dir:   filepath.Join(util.CacheDir(), "fetch", "cisco", "json"),
+		retry: 3,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "cisco-json <CISCO Client Key> <CISCO Client Secret>",
```

Comment:
I'm just a bit curious why CISCO is all capital. "Client" and "Key" are both capital at first letters only. "Cisco" is better ?

Reply:
fixed: https://github.com/MaineK00n/vuls-data-update/compare/bc547a7cc39eb02cea5d7206f116ea5a1ec2b23a..c1d4b868495f97ddc95837f71ac666cd35f19a73